docs/wiki/deployment/process-auditing.md

```diff
@@ -175,10 +169,44 @@ osqueryi --audit_allow_config=true --audit_allow_sockets=true --audit_persist=tr
 
 If you would like to debug the raw audit events as `osqueryd` sees them, use the hidden flag `--audit_debug`. This will print all of the RAW audit lines to osquery's `stdout`.
 
-## User events
+## User event auditing with Audit
 
 On Linux, a companion table called `user_events` is included that provides several authentication-based events. If you are enabling process auditing it should be trivial to also include this table.
 
+## Linux process auditing using BPF
+
+When osquery is running on a recent kernel (>= 4.18), the BPF eventing framework can be used. This event publisher needs to monitor for more system calls to reach feature parity with the Audit-based tables. For this reason, enabling BPF will also enable both the `bpf_process_events` and `bpf_socket_events` tables.
+
+In order to start the publisher and enable the subscribers, the following flags must be passed: `--disable_events=false --enable_bpf_events=true`. The `--verbose` flag can also be extremely useful when setting up the configuration for the first time, since it emit more debug information when something fails.
+
+The BPF framework will make use of a perf event array and several per-cpu maps in order to receive events and correctly capture strings and buffers. These structures can be configured using the following command line flags:
+ - **bpf_perf_event_array_exp**: size of the perf event array, as a power of two
+ - **bpf_buffer_storage_size**: how many slots of 4096 bytes should be available in each memory pool
+
+Memory usage depends on both
+ 1. How many processors are currently online
+ 2. How many processors can be added by hotswapping
+
+The BPF event publisher uses 6 memory pools, grouping system calls in order to evenly distribute memory usage. Not counting the internal maps used to merge sys_enter/sys_exit events (the size for these maps is rather small), memory usage can be easily estimated with the following formula:
+
+```
+buffer_storage_bytes = memory_pool_count * (bpf_buffer_storage_size * 4096) * possible_cpu_count
+```
+```
+perf_bytes = (2 ^ bpf_perf_event_array_exp) * online_cpu_count
+```
+
+The cpu count numbers can be read from the /sys folder:
```

Comment:
Do you mind double checking the documentation for values like `/sys`, which should be within back-ticks? For example, I recommend back-ticking `sys_enter` and similar in the above paragraphs too.